### ISSUE TITLE
[Bug] Setting value for minY to -1 stops ore from spawning, even when maxY is not -1

### ISSUE THREAD

**Issue by Goldenlion5648** [EwyBoy/OreTweaker#79]
###### **Minecraft Version**
> 1.18.2

###### **Forge Version**
> 40.1.30

###### **Mod Version**
> OreTweaker-1.18.2-3.3.0.jar

###### **Describe the bug**
> Setting minY to -1 makes ore not spawn (even when maxY is greater than -1). I think the ore should only be disabled when both values are -1.

###### **How to reproduce**
> set minY to -1, and maxY to a value such as 9 for an ore. Ore does not spawn as a result

###### **Screenshots / Images**
The ore in this case is lava replacing grass blocks
minY at -1
![minY at -1](https://user-images.githubusercontent.com/32916571/180120560-f904bafa-5c25-45e6-94b7-23a0e1c6b300.png)

minY at 0
![minY at 0](https://user-images.githubusercontent.com/32916571/180120533-3402cf40-3b78-46f7-a9d2-dcd5bf9f5b06.png)

**Comment by EwyBoy** [comment]
You are right, there has probably been an oversight in the code when porting to 1.17 when the worlds expanded to go below 0. I think the old value for disabling ores where to set the values to -1.

Thank you for the detailed bug report.
